src/applicationcomposer/messageHandlers/emitTelemetryMessageHandler.ts

```diff
@@ -61,6 +62,9 @@ export function emitTelemetryMessageHandler(message: EmitTelemetryMessage) {
             case 'CLOSE_WFS':
                 sendCloseWfs(parsedData as AppcomposerCloseWfs)
                 return
+            case 'TEMPLATE_OPENED':
+                sendResourceCounts(parsedData as AppcomposerResourceCount)
```

Comment:
why doesn't this emit the existing `appcomposer_openTemplate` event?

Reply:
Despite the naming, they are two separate events. The existing `appcomposer_openTemplate` event requires auth data that this event does not have access to

Reply:
@justinmk3 Could I get a re-review of this PR?

Reply:
I still don't understand why a new event is being added. `appcomposer_resourceCount` is not an "event" it's just data. Is there an event this data can be attached to?